src/ObjectList.php

```diff
@@ -357,4 +359,19 @@ public function sum(callable $valueProvider, $startValue = null)
             return $total + $valueProvider($item);
         }, $startValue);
     }
+
+    /**
+     * @param callable $filter
+     * @return mixed
+     */
+    public function get(callable $filter)
```

Comment:
Yeah, and `get` in this case requires the value to exist. `find` would (imo) imply that the value can not exist.